Headers/IOKit/bluetooth/IOBluetoothACPIMethods.h

```diff
@@ -0,0 +1,112 @@
+/*
+ *  Released under "The GNU General Public License (GPL-2.0)"
```

Comment:
Maybe something less restricted than GPLv2? I do not think anybody would really use that for non-OSS stuff, but I think BSD license (e.g. BSD 2-clause or BSD 3-clause) are more fit for macOS, which is a BSD system itself.

Reply:
I switched to the BSD 3-clause license.